Automattic/wp-calypso
# Theme Showcase: Show theme demo site in the preview for third-party themes

#### Proposed Changes

This PR checks whether the theme preview is generated from the wpcom site preview endpoint. If that's not the case, then assume that the demo URL is external and embedded it as-is to the site preview iframe.

![Screen Shot 2023-01-13 at 8 55 17 AM](https://user-images.githubusercontent.com/797888/212212366-52194fd0-5466-48dc-b278-05c2a6155874.png)


#### Testing Instructions



* Head to the logged-in Theme Showcase `/themes/${site_slug}`. If using calypso.live, the flag `themes/showcase-i4/details-and-preview` is required.
* Search for a third-party theme, such as "Astra" or "Jaxon".
* Ensure that its theme preview shows the theme demo site.
* Ensure that for first-party themes, we are still showing the preview via the wpcom site preview endpoint.

#### Pre-merge Checklist
